.jscsrc

```diff
@@ -38,7 +38,7 @@
    "disallowYodaConditions": true,
    "maximumLineLength": 120,
    "requireBlocksOnNewline": true,
-   "requireCamelCaseOrUpperCaseIdentifiers": true,
+   "requireCamelCaseOrUpperCaseIdentifiers": false,
```

Comment:
Main reasons I disabled them were _request and  (dangling underscores) and "per_page" from GitHub's API (camelcase or upper).